src/plugins/intel_npu/src/utils/src/zero/CMakeLists.txt

```diff
@@ -37,14 +37,20 @@ ov_developer_package_export_targets(TARGET openvino::npu_zero_utils
                                     INSTALL_INCLUDE_DIRECTORIES
                                         $<BUILD_INTERFACE:${NPU_UTILS_SOURCE_DIR}/include>)
 
-ov_developer_package_export_targets(TARGET level-zero-ext)
+ov_developer_package_export_targets(
+    TARGET level-zero-ext
+    INSTALL_INCLUDE_DIRECTORIES
+        $<TARGET_PROPERTY:level-zero-ext,INTERFACE_INCLUDE_DIRECTORIES>
```

Comment:
Please reduce the list of the exported files (we need to collect the required for the build files only)
```suggestion
        $<TARGET_PROPERTY:level-zero-ext,INTERFACE_INCLUDE_DIRECTORIES>
```
![image](https://github.com/user-attachments/assets/3a3be8dd-9575-48e2-8716-ca27a989d98d)

Reply:
Removed and just retaining required files.